src/components/PrimaryNav.tsx

```diff
@@ -90,6 +93,19 @@ const PrimaryNav = (): JSX.Element => {
                   )}
                 </Menu.Item>
 
+                <Menu.Item>
+                  {({ active }) => (
+                    <button
+                      className={`${
+                        active ? "bg-gray-100 text-gray-700" : "text-gray-900"
+                      } group flex w-full items-center rounded-md px-[20px] py-[6px] text-sm`}
```

Comment:
Could you please not use this template? `px-[20px]` you could use like that`px-5` #267

Reply:
sure

Reply:
you can check more things [here](https://flowbite.com/tools/tailwind-cheat-sheet/)